Solutions/McAfee Network Security Platform/Data Connectors/McAfeeNSP.json

```diff
@@ -0,0 +1,127 @@
+                                "parameters": {
+                                    "linkType": "InstallAgentOnLinuxNonAzure"
+                                },
+                                "type": "InstallAgent"
+                            }]
+                        }
+                    ]
+                },
+                "type": "InstructionStepsGroup"
+            }]
+        },
+        {
+            "instructions": [{
+                "parameters": {
+                    "title": "Choose where to install the Windows agent:",
+                    "instructionSteps": [{
+                            "title": "Install agent on Azure Windows Virtual Machine",
+                            "description": "Select the machine to install the agent on and then click **Connect**.",
+                            "instructions": [{
+                                "parameters": {
+                                    "linkType": "InstallAgentOnVirtualMachine"
+                                },
+                                "type": "InstallAgent"
+                            }]
+                        },
+                        {
+                            "title": "Install agent on a non-Azure Windows Machine",
+                            "description": "Download the agent on the relevant machine and follow the instructions.",
+                            "instructions": [{
+                                "parameters": {
+                                    "linkType": "InstallAgentOnNonAzure"
+                                },
+                                "type": "InstallAgent"
+                            }]
+                        }
+                    ]
+                },
+                "type": "InstructionStepsGroup"
+            }]
+        },
+        {
+            "title": "2. Configure McAfee® Network Security Platform event forwarding",
+            "description": "Follow the configuration steps below to get McAfee® Network Security Platform logs into Azure Sentinel.\n1. [Follow these instructions](https://docs.mcafee.com/bundle/network-security-platform-10.1.x-product-guide/page/GUID-E4A687B0-FAFB-4170-AC94-1D968A10380F.html) to forward alerts from the Manager to a syslog server.\n2. Add a syslog notification profile, [more details here](https://docs.mcafee.com/bundle/network-security-platform-10.1.x-product-guide/page/GUID-5BADD5D7-21AE-4E3B-AEE2-A079F3FD6A38.html). This is mandatory. While creating profile, to make sure that events are formatted correctly, enter the following text in the Message text box:\n\t\t<SyslogAlertForwarderNSP>:|SENSOR_ALERT_UUID|ALERT_TYPE|ATTACK_TIME|ATTACK_NAME|ATTACK_ID\n\t\t|ATTACK_SEVERITY|ATTACK_SIGNATURE|ATTACK_CONFIDENCE|ADMIN_DOMAIN|SENSOR_NAME|INTERFACE\n\t\t|SOURCE_IP|SOURCE_PORT|DESTINATION_IP|DESTINATION_PORT|CATEGORY|SUB_CATEGORY\n\t\t|DIRECTION|RESULT_STATUS|DETECTION_MECHANISM|APPLICATION_PROTOCOL|NETWORK_PROTOCOL|"
+        }
+    ]
+}
```

Comment:
Please add a Metadata object within this file. See [https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_Syslog_template.json#L110](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_Syslog_template.json#L110) for further detail.